Sources/Fullscreen/EmptyView/EmptyView.swift

```diff
@@ -161,6 +153,31 @@ public class EmptyView: UIView {
         ])
     }
 
+    public override func layoutSubviews() {
+        super.layoutSubviews()
+
+        // We only want to lay out once
+        guard hasLayedOut == false else {
+            return
+        }
+
+        let slice = frame.width / 8
+
+        // We resposition the shapes after the EmptyView itself has layed out its frame.
+        // At this point we will have its size even if we use constraints to lay it out.
+        triangle.center = CGPoint(x: slice, y: frame.height - (sizeOfTriangle.height / 2))
+        circle.center = CGPoint(x: slice * 3, y: frame.height - (sizeOfCircle.height / 2))
+        roundedSquare.center = CGPoint(x: slice * 5, y: frame.height - (sizeOfRoundedSquare.height / 2))
+        rectangle.center = CGPoint(x: slice * 7, y: frame.height - (sizeOfSquare.height / 2))
```

Comment:
The `slice` system is nice, but there is a possibility that an object will start outside the width of the screen if the object close to the edge is big enough. Just sayin'.. I don't think it will be a problem since we actually decide how large the objects are, but it's worth a mention.